ios/FluentUI/People Picker/ContactCollectionView.swift

```diff
@@ -0,0 +1,161 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+// MARK: ContactCollectionViewDelegate
+
+public protocol ContactCollectionViewDelegate: AnyObject {
+    func didTapOnContactViewAtIndex(index: Int, personaData: PersonaData)
```

Comment:
This function needs to be exposed to objc (@objc).

Reply:
Also, in general, delegate methods are optional.

Reply:
We want to expose the functions in the protocol to objc because other classes written in objc may want to be the delegate right?

Reply:
right